lib/kubeclient/common.rb

```diff
@@ -253,6 +240,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace, container: nil, previous: false)
```

Comment:
> According to the Kubernetes API docs, "Return previous terminated container logs" if true.
> 
> I believe it's for a case where a container terminates in a pod, and is automatically restarted. This is a mechanism to get the logs of the terminated container, rather than the currently running container.

alright, please either link in the readme to the k8s doc, or just add it inline - whatever you prefer.
I would like the readme to be as clear as possible to the gem users.
the PR LGTM,
@simon3z please review, especially the watch part. (and you had a comment in the linked issue on k8s functionality regarding logs, please explain it, in case your comment applies to this patch)